Makefile

```diff
@@ -39,8 +39,14 @@ fresh_vc:
 	git clone --depth 1 --quiet https://github.com/vlang/vc
 
 fresh_tcc:
+ifdef WIN32
+	rm -rf /var/tmp/tcc/
+	git clone --depth 1 --quiet https://github.com/vlang/tccbin_win /var/tmp/tcc
```

Comment:
I think WIN32 platforms do not have /var/tmp/tcc but I may be wrong for MSYS, not sure.

Reply:
on MSYS2 it works => current V master downloads linux tcc out ther :-)

Reply:
Cool.